src/axel_f/excel/validate.cljc

```diff
@@ -0,0 +1,28 @@
+(ns axel-f.excel.validate)
+
+(defn presence*
+  "Validate and return if not null given value and throw an Exception with given message (default is 'Argument is required')."
```

Comment:
This one is a bit hard to understand - does it mean 'Validate and return given value if not null, and throw an exception with given message'?

Reply:
exactly as you phrase it, thanks!